templates/vhosts80/rpm.conf

```diff
@@ -1,3 +1,2 @@
-Alias /pulp/repos /var/www/pub/yum/http/repos
```

Comment:
This one alias is no longer needed.  Instead the streamer wsgi app handles it and decides whether to serve the content or fetch it from upstream.